mrp_unbuild_advanced/models/mrp_unbuild.py

```diff
@@ -31,6 +31,7 @@ class MrpUnbuild(models.Model):
         states={"draft": [("required", False)], "done": [("required", True)]},
     )
     shift_stop_time = fields.Float()
+    scheduled_time = fields.Float(default=8)
```

Comment:
This field is missing in views, isn't it?

Reply:
It is, fixed